ReSharper.FSharp/src/FSharp/FSharp.Common/src/Util/FSharpSymbolUtil.fs

```diff
@@ -30,6 +32,55 @@ let isEnumMember (field: FSharpField) =
     | Some entity -> entity.IsEnum
     | _ -> false
 
+let formatMfv (mfv: FSharpMemberOrFunctionOrValue) (displayContext: FSharpDisplayContext) addParameterNames =
```

Comment:
Do you think we could apply this function partially in some of the features? If yes, reversing the parameter list would make it better.

Reply:
fixed